### ISSUE TITLE
JavaScript error during the update to 1.0.1

### ISSUE THREAD

**Issue by mitio** [careteditor/caret#29]
A JS error occurred during the update to the app's latest version, 1.0.1. The update seems to otherwise complete successfully. Here's what happened:

I was using Night mode. I started the app, no files were open. I was looking for the "Enter license" menu in Help, but found none. I clicked on "Report an issue" by accident, the issues page on GitHub opened in a browser. Went back to the app and saw that the "update available" bar had appeared. I clicked restart. The app closed and the following error popped up:

```
A JavaScript error occurred in the main process

Uncaught Exception:
TypeError: autoUpdater.quitAndUpdate is not a function
    at EventEmitter.<anonymous> (/Applications/Caret.app/Contents/Resources/app.asar/main/AutoUpdater.js:5:681)
    at emitOne (events.js:82:20)
    at EventEmitter.emit (events.js:169:7)
```

I clicked "OK" and after a bit a new version of the app started. It seems to be working okay for now. It reports it's version 1.0.1. The "Enter license" menu was now present and I was able to successfully activate my license.

**Comment by erusev** [comment]
This is resolved in 1.0.1, it should no longer occur.

Thanks a lot for the great report and for purchasing a license.

**Comment by mitio** [comment]
I suspected it was a one-time problem. Closing this issue.
